web/src/features/charts/BreakdownChart.tsx

```diff
@@ -38,6 +39,17 @@ function BreakdownChart({
 
   const titleDisplayMode = displayByEmissions ? 'emissions' : 'electricity';
   const titleMixMode = mixMode === Mode.CONSUMPTION ? 'origin' : 'production';
+
+  const hasEnoughDataToDisplay = datetimes?.length > 2;
```

Comment:
Should this not be:
```suggestion
  const hasEnoughDataToDisplay = datetimes?.length >= 2;
```
Or do we only want to show it when we have more than two datapoints?

Reply:
We currently expect these things to be available, so we kinda do need more than two:
```js
  const startTime = datetimes?.at(0);
  const lastTime = datetimes?.at(-1);
  const interval = datetimes?.at(-2);
```

I briefly looked into making it work with just one or two datapoints, but that was a lot more work that I didn't want to get started on now 😅